models/migrations/migrations.go

```diff
@@ -554,6 +554,9 @@ var migrations = []Migration{
 	NewMigration("Add combined Index to issue_user.uid and issue_id", v1_22.AddCombinedIndexToIssueUser),
 	// v284 -> v285
 	NewMigration("Add ignore stale approval column on branch table", v1_22.AddIgnoreStaleApprovalsColumnToProtectedBranchTable),
+	// v285 -> v286
```

Comment:
```suggestion
	// v285 -> v287
```

well it's either to counter increments or you combine the migratons